### DISCUSSION TITLE
Detectable by other Anticheats?

### DISCUSSION THREAD

**Discussion by stickyystick** [34736384/genshin-fps-unlock#373] [Q&A] [answered]
Hi if i have this fps unlocker on my computer will it get flagged by other games? For example if I want to play fortnite and i have this app on my computer will fortnite see it as a third party program and punish me for it? This is because the fortnite anticheat scans the computer for any suspicious files and i do not wish to be banned from other games,
Thanks!

**Comment by 34736384** [comment, marked as answer]
In theory, it shouldn't cause any issues with other anti-cheats, but I don't have a guarantee for it.

**Comment by MenonPlayz** [comment]
Definitely not an issue for me, no matter the game. Easy Anticheat doesnt complain, which is what Fortnite is using
